packages/math-input/src/components/keypad/stateful-keypad.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import {action} from "@storybook/addon-actions";
+import * as React from "react";
+
+import StatefulKeypad from "./stateful-keypad";
+
+export default {
+    title: "Stateful v2 Keypad",
```

Comment:
This should be `Perseus/Editor/Stateful v2 Keypad` so that it goes into the right place in the Storybook "menu", I think.